src/java.base/share/classes/java/lang/reflect/ProxyGenerator.java

```diff
@@ -453,7 +453,7 @@ protected ClassLoader getClassLoader() {
      * class file generation process.
      */
     private byte[] generateClassFile() {
-        visit(V16, accessFlags, dotToSlash(className), null,
+        visit(V17, accessFlags, dotToSlash(className), null,
```

Comment:
Alternatively, we can use the value of the system property `java.class.version` to prepare for future class file version change.  But this requires parsing it to extract the major version.

I'm okay with this simple approach and remember to update it if the class file version is bumped.

Reply:
It seems reasonable. When I fix another related fail, I can tweak this part.